contracts/sol6/Dao/KyberDAO.sol

```diff
@@ -745,8 +742,6 @@ contract KyberDAO is IKyberDAO, EpochUtils, ReentrancyGuard, Utils5, DaoOperator
         require(cInPrecision <= POWER_128, "validateParams: c is high");
 
         require(tInPrecision <= POWER_128, "validateParams: t is high");
-
-        return true;
```

Comment:
this is a public function for our operation team to check if params are valid. Not sure if we should remove return here?

Reply:
yes
lets make it internal